ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpImplTreeBuilder.fs

```diff
@@ -230,8 +235,15 @@ type FSharpImplTreeBuilder(lexer, document, decls, lifetime, path, projectedOffs
             match x.ContinueMemberDecl(typeMember.Range) with
             | ValueSome(mark) -> mark
             | _ ->
-                let mark = x.MarkXmlDocOwner(typeMember.XmlDoc, null, typeMember.Range)
-                x.ProcessAttributeLists(typeMember.Attributes)
+                let mark =
+                    match typeMember with
+                    | SynMemberDefn.LetBindings _ ->
+                        x.AdvanceToXmlDocOwner(typeMember.XmlDoc, null, typeMember.Range)
+                        x.Mark()
```

Comment:
Why do you call separate `Advance*` and `Mark` method instead of single `MarkXmlDocOwner`, as prior the change? Is there something that got broken in `MarkXmlDocOwner` in this PR?

Reply:
The MarkXmlDocOwner logic, in particular, marks the documentation itself and binds it to the declaration, which is not always required. In some cases, you just need to reach a certain group of declarations without marking the first declaration XmlDoc.

Reply:
I see. It would probably be better to either add another helper method or add a parameter to `MarkXmlDocOwner` to alter its behaviour. With the current logic a comment about marking the declarations/bindings group would really help.